src/main/java/org/sagebionetworks/bridge/validators/HealthDataRecordValidator.java

```diff
@@ -86,17 +86,17 @@ public void validate(Object object, Errors errors) {
                 errors.rejectValue("metadata", Validate.WRONG_TYPE);
             }
 
-            // schema ID is non-null and non-empty
-            if (StringUtils.isBlank(record.getSchemaId())) {
-                errors.rejectValue("schemaId", Validate.CANNOT_BE_BLANK);
+            // If schema ID is specified, it can't be blank.
+            if (record.getSchemaId() != null && StringUtils.isBlank(record.getSchemaId())) {
+                errors.rejectValue("schemaId", "can't be blank if specified");
```

Comment:
really trivial but I think "cannot" (which is more formal) is appropriate in error messages.